src/cascadia/TerminalSettingsEditor/ProfileViewModel.cpp

```diff
@@ -115,6 +115,22 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
                     Opacity(1.0);
                 }
             }
+            else if (viewModelProperty == L"AntialiasingMode")

```

Comment:
Just double checking... the property is "Antialiasing" (with the "A" of "aliasing") being lowercase while the Notification string is "AntiAliasing" (with a capital "A")?

It may very well have already been like that and you just copy/pasted... but I'm just checking as it stood out to me.

Reply:
Yeah, I was actually really surprised this worked haha.